### ISSUE TITLE
Files not compiling

### ISSUE THREAD

**Issue by meshakeeb** [moyus/laravel-mix-nunjucks#8]
Hi,

The files are not compiling if a partial is change. Can you please look into this.

Thanks in advance and thanks for great extension

**Comment by florianbouvot** [comment]
@moyus did you plan to update your extension?

**Comment by moyus** [comment]
Is there any bug with Laravel Mix 6?

**Comment by florianbouvot** [comment]
@moyus it works for me but I need to use `--force` to install your extension

![Capture d’écran 2021-02-25 à 09 10 27](https://user-images.githubusercontent.com/2061540/109122738-535f3b00-7749-11eb-8992-5a5b9e824f5a.png)

**Comment by moyus** [comment]
I will fix it tonight, hang on there!

**Comment by lemonsprites** [comment]
![image](https://user-images.githubusercontent.com/55780017/164968237-40d3b92e-ab32-41a0-a867-cc66aaf26b55.png)

Its successful to compile other file, but no nunjuck file get compiled here, I have followed the configuration from the repo but it still doesn't work for me, and I have tried the repo test folder and the result is the same, files aren't compiled.

Here my config file
```
//> webpack.mix.js


let mix = require('laravel-mix');

require('laravel-mix-nunjucks');

mix.njk('src/views/', 'views/', {
    ext: '.html',
    block: 'konten',
    marked: null,
    watch: true,
    manageEnv: (nunjucks) => {},
})

mix.postCss('src/css/style.css', 'lemonflex.css', [
    require('postcss-import'),
    require('tailwindcss/nesting')('postcss-nested'),
    require('tailwindcss')({
        mode: 'jit',
        content: [
            './src/**/*.{html,js}',
            './dist/**/*.{html,js}'
        ],
        theme: {
            extend: {
                fontFamily: {
                    'sans': ['Noto Sans', 'Heebo', 'sans-serif']
                },
            },
        },
    }),
    require('autoprefixer'),
])

mix.js('src/main.js', 'lemonflex.js')


mix.options({
    fileLoaderDirs: {
        fonts: ''
    },
})
    .disableNotifications()
    .setPublicPath("dist");
```

is someone know why it haven to me?
btw,  I just updated Node.js to the latest version before installing this ext. (v16.13.2  => v16.14.2).

**Comment by moyus** [comment]
@lemonsprites please check whether nunjuck files have been compiled into your output folder, they may not show up in the console but do have been compiled.

**Comment by lemonsprites** [comment]
Hey thanks for the reply, @moyus . Yeah you right i found it, but it compiled when i watch the file (```npx mix watch```), and its not compiled when i run for single compile (```npx mix```). :smiley:

**Comment by Niksac** [comment]
@moyus I got the same issue as @lemonsprites .
When I run `npm run mix` or `npm run mix --production`, the Nunjucks files will not compile.
When I run `npm run mix`, the files will compile as expected.

Running version 1.1.0

**Comment by moyus** [comment]
I will update a new version to fix this problem soon, hang on there!

**Comment by moyus** [comment]
@lemonsprites I have published a beta version `laravel-mix-nunjucks@1.2.0-beta.1`, could you please help me test whether it works now?

**Comment by lemonsprites** [comment]
thanks @moyus , its work properly, and now i can see the compiled file in list its mean a lot for me.😆 
![image](https://user-images.githubusercontent.com/55780017/170878892-1d311c47-9e04-43c9-a303-7ad6daf64e48.png)
